src/Core/Animation/KeyFramedValue.hpp

```diff
@@ -58,8 +58,7 @@ class RA_CORE_API KeyFramedValueBase
  *
  * \note There is always at least one keyframe defined.
  */
-template <typename VALUE_TYPE>
```

Comment:
A titre perso je préfère garder la séparation entre le type template et le reste de la déclaration.

Reply:
For the record 
> "Personally, I prefer to keep the template type separate from the rest of the declaration."

Ok, my concern raises when I have some `using` def with `template`. But there is no way currently to have different behavior of the template break for that in clang-format. But fine to stick with the always break.

I'll split other proposed style changes to check if there meaningful.

```{.cpp}
    template <typename Port>
    using PortPtr = std::shared_ptr<Port>;
    template <typename Port>
    using PortRawPtr = typename PortPtr<Port>::element_type*;

    template <typename Type>
    using PortInPtr = PortPtr<PortIn<Type>>;
    template <typename Type>
    using PortInRawPtr = typename PortInPtr<Type>::element_type*;

    template <typename Type>
    using PortOutPtr = PortPtr<PortOut<Type>>;
    template <typename Type>
    using PortOutRawPtr = typename PortOutPtr<Type>::element_type*;
````
vs
```{.cpp}

    template <typename Port> using PortPtr    = std::shared_ptr<Port>;
    template <typename Port> using PortRawPtr = typename PortPtr<Port>::element_type*;

    template <typename Type> using PortInPtr    = PortPtr<PortIn<Type>>;
    template <typename Type> using PortInRawPtr = typename PortInPtr<Type>::element_type*;

    template <typename Type> using PortOutPtr    = PortPtr<PortOut<Type>>;
    template <typename Type> using PortOutRawPtr = typename PortOutPtr<Type>::element_type*;
```

Reply:
You can check 94deb27 concerning oneliner.

also I vote to switch to c++17 (or c++11, but not Cpp11 since "Cpp11 is a deprecated alias for Latest" [clang-format-doc](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#standard)

Reply:
I prefer :
```c++
template <typename Port>
using PortPtr = std::shared_ptr<Port>;

template <typename Port>
using PortRawPtr = typename PortPtr<Port>::element_type*;
```